gradients.json

```diff
@@ -731,5 +731,13 @@
     {
         "name": "Brady Brady Fun Fun",
         "colors": ["#00c3ff", "#ffff1c"]
+    },
+    {
+        "name": "Snapchat",
+        "colors": ["#fffc00", "#ffffff"]
+    },
+    {
+        "name": "Snapchat",
```

Comment:
@teded32 Snapchat is repeated twice. Probably should change it to something else.